ports/perfetto/CMakeLists.txt

```
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.8.0)
+project(perfetto)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+include(CMakePackageConfigHelpers)
+
+include_directories(sdk)
+add_library(perfetto sdk/perfetto.cc)
+
+if (MSVC)
+    target_compile_options(perfetto PRIVATE "/bigobj")
+    target_compile_options(perfetto PRIVATE "/permissive-")
+    target_compile_definitions(perfetto PRIVATE WIN32_LEAN_AND_MEAN NOMINMAX)
+endif (MSVC)
+
+install(
+    TARGETS perfetto
+    EXPORT unofficial-perfetto-config
+    LIBRARY DESTINATION lib
+    ARCHIVE DESTINATION lib
+)
+
+install(FILES sdk/perfetto.h DESTINATION include/)
+
+install(
+    EXPORT unofficial-perfetto-config
+    NAMESPACE unofficial::
```

The suggested pattern is:
```suggestion
    NAMESPACE unofficial::perfetto::
```